h/crossmemory.h

```diff
@@ -147,12 +147,15 @@
 #define RC_CMS_MODREG_FAILED                96
 #define RC_CMS_MAX_RC                       96
 
+#define CMS_CONFIG_PARM_MAX_NAME_LENGTH   72
+#define CMS_CONFIG_PARM_MAX_VALUE_SIZE    128
+
 extern const char *CMS_RC_DESCRIPTION[];
 
 ZOWE_PRAGMA_PACK
 
 typedef struct CrossMemoryServerName_tag {
-  char nameSpacePadded[16];
+  char nameSpacePadded[CMS_CONFIG_PARM_MAX_NAME_LENGTH];
```

Comment:
We cannot change this size; 72 chars is just a limit set for cross-memory server parameters' name, not cross-memory server names. This change would break lots of things.

Reply:
So, for example, you can have a parameter name up to 72 chars and its value up to 128 chars:
```
ZWES.MY_CUSTOM_LONG_PARM=long_long_value_1234567890
```
This has nothing to do with cross-memory server names though.

Reply:
Yes, I have realized that, my fault. Sorry for confusion.